### ISSUE TITLE
딥러닝

### ISSUE THREAD

**Issue by jangjusung** [jangjusung/jusung-python#33]
from sklearn.preprocessing import LabelEncoder
import pandas as pd
import matplotlib.pyplot as plt
import seaborn as sns
import numpy as np
import tensorflow as tf
import os

np.random.seed(3)
tf.random.set_seed(3)
from tensorflow.keras.models import Sequential
from tensorflow.keras.layers import Dense
from tensorflow.keras.callbacks import ModelCheckpoint

df_pre=pd.read_csv("datasets\wine.csv",header=None)
df=df_pre.sample(frac=1)
df.head()

data=df.values
X=data[:,0:12].astype("float")
Y=data[:,12]

#Y의 값 변환
e=LabelEncoder()
e.fit(Y)
target=e.transform(Y)

train_input, test_input, train_target, test_target=\
train_test_split(X,target,test_size=0.3,stratify=target)

model=Sequential()
model.add(Dense(24,input_dim=12, activation='relu'))
model.add(Dense(10,activation='relu'))
model.add(Dense(1,activation='sigmoid'))
model.summary()

model.compile(loss="binary_crossentropy",
             optimizer="adam",metrics=["accuracy"])
model.fit(train_input, train_target, epochs=50,batch_size=10)
model.compile(loss="binary_crossentropy",
             optimizer="adam",metrics=["accuracy"])
model.fit(train_input, train_target, epochs=50,batch_size=10)

model.evaluate(test_input, test_target)

MODEL_DIR='./model/'
if not os.path.exists(MODEL_DIR):
    os.mkdir(MODEL_DIR)

modelpath="./model/{epoch:02d}-{val_loss:4f}.hdf5"
checkpointer=ModelCheckpoint(filepath=modelpath,monitor='val_loss', 
                             verbose=1,save_bst_only=True)

model.fit(X,Y,validation_split=0.2, epochs=200, batch_size=200,verbose=0,callbacks=[checkpointer])
